tokio/src/runtime/tests/task.rs

```diff
@@ -45,6 +47,41 @@ impl Drop for AssertDrop {
     }
 }
 
+#[test]
+fn drop_tasks_with_reference_cycle() {
```

Comment:
This does not use any Tokio internals, so it does not need to be inside `src/`. It should be in `tokio/tests/` instead.

Reply:
I'll move it for you.

Reply:
Oh thanks I wasnt aware of this but will keep it in mind for the future.

Reply:
No worries. Thanks a lot for the PR!